load.php

```diff
@@ -6,13 +6,17 @@
 	define( 'WP_CACHE', true );
 }
 
+if ( ! defined( 'HM_ENV_TYPE' ) ) {
+	define( 'HM_ENV_TYPE', 'local' );
+}
+
 // Load the platform as soon as WP is loaded.
 $GLOBALS['wp_filter']['enable_wp_debug_mode_checks'][10]['hm_platform'] = array(
 	'function' => __NAMESPACE__ . '\\bootstrap',
 	'accepted_args' => 1,
 );
 
-if ( class_exists( 'HM\\Cavalcade\\Runner\\Runner' ) && get_config()['cavalcade'] ) {
+if ( class_exists( 'HM\\Cavalcade\\Runner\\Runner' ) && get_config()['cavalcade'] && HM_ENV_TYPE !== 'local' ) {
```

Comment:
In cases when we want to run cavalcade locally, I'm not sure if this makes sense.

Reply:
It doesn't stop cavalcade running or loading though. Just doesn't load the AWS logging code.

Reply:
Ah ok - I think either way we should make it configurable, as, for one, I need to be able to develop locally and test this to build it!

Reply:
True! You could define `HM_ENV_TYPE` early in your local config for that :)